dotcom-rendering/src/web/components/ContainerTitle.tsx

```diff
@@ -60,13 +50,22 @@ const descriptionStyles = (fontColour?: string) => css`
 		color: ${text.primary};
 		text-decoration: none;
 	}
+`;
+
+const bottomMargin = css`
+	margin-bottom: ${space[2]}px;
+`;
+
+const marginStyles = css`
+	margin-left: 0;
 	${between.tablet.and.leftCol} {
 		margin-left: 10px;
 	}
 
-	${until.leftCol} {
-		margin-bottom: ${space[4]}px;
+	${from.leftCol} {
+		margin-left: 0;
 	}
```

Comment:
Is this not redundant? Assuming the `between.tablet.and.leftCol` no longer applies

Reply:
You're right. Removed